src/client/content_cao.cpp

```diff
@@ -1304,7 +1304,8 @@ void GenericCAO::updateAttachments()
 			player->isAttached = true;
 		}
 		
-		if (m_env->getGenericCAO(m_env->attachement_parent_ids[getId()]) && m_env->getGenericCAO(m_env->attachement_parent_ids[getId()])->m_is_local_player) {
+		if (m_env->getGenericCAO(m_env->attachement_parent_ids[getId()]) 
+			&& m_env->getGenericCAO(m_env->attachement_parent_ids[getId()])->m_is_local_player) {
```

Comment:
As this line is a continuation of the previous line, this has to be indented by two tab-spaces to adhere to code-style.

Reply:
Please move the value of `m_env->getGenericCAO(m_env->attachement_parent_ids[getId()])` into a variable